java-components/management-console/src/main/java/com/redhat/hacbs/management/model/DependencySet.java

```diff
@@ -25,6 +27,7 @@ public class DependencySet extends PanacheEntity {
     @Column(nullable = false)
     public String type;
 
+    @JsonManagedReference
     @OneToMany(fetch = FetchType.EAGER, mappedBy = "dependencySet", cascade = CascadeType.ALL)
```

Comment:
@stuartwdouglas I'm not entirely sure I've got the correct json annotations here (and in IdentifiedDependency below)

Reply:
Why do you need this, is there something in ImageDTO that is not needed?

In general I am not a fan of using database backed types in JSON endpoints. It is really easy to cause security issues as Jackson can follow relationships and include more data than you expect.

It can also be possible to do the opposite, and update different entities to what you expect if a POST includes updates to related entities.

Reply:
Yeah looks like ImageDTO with `analysisFailed` and then another call of `getDependencySet` to get `DependencySetDTO`